workers/unity/Assets/Playground/Scripts/OneTimeInitialisation.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
```

Comment:
Please rename this class to `OneTimeInitialization` to adhere to American English.

Reply:
This was added in a previous PR though and I'd rather not make driveby changes this far from the focus of the PR